src/components/SecurityContainer/SecurityContainer.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../base.scss';
+
+.security-container {
+  padding: 1rem 0;
+  margin-bottom: 2rem;
+  border-bottom: 1px solid $border-color;
+
+  .scheme {
+    display: inline-block;
+    padding: .4rem;
+    margin-left: .5rem;
+    background-color: $navigation-background;
+  }
+
+  .flow-type {
+    padding: 1rem 0;
+
+    & + .flow-type {
```

Comment:
When there are multiple flows provided, which is not possible with our current fixtures, but is possible with the spec.